{{cookiecutter.github_repository_name}}/src/{{cookiecutter.module_name}}/serializers.py

```diff
@@ -124,6 +125,10 @@ def validate(self, validated_data):
 
     def delete(self):
         company = self.validated_data['company']
+        purge = self.validated_data['purge']
```

Comment:
As per the above, then change this to `self.validated_data.get('purge', False)`.

And the `if purge` can be more explicit as `if purge is True:`